corehq/apps/accounting/tests/test_forms.py

```diff
@@ -99,3 +108,105 @@ def test_transfer_credit_without_credit(self):
             credit_line.balance
             for credit_line in CreditLine.get_credits_for_invoice(self.invoice)
         ))
+
+
+class TestSubscriptionForm(BaseAccountingTest):
+
+    def setUp(self):
+        super(TestSubscriptionForm, self).setUp()
+
+        self.domain = Domain(
+            name="test-sub-form",
+            is_active=True
+        )
+        self.domain.save()
+        self.domain2 = Domain(
+            name="test-sub-form-2",
+            is_active=True
+        )
+        self.domain2.save()
+
+        self.web_user = WebUser.create(
+            self.domain.name, generator.create_arbitrary_web_user_name(), 'testpwd'
+        )
+
+        self.account = BillingAccount.get_or_create_account_by_domain(
+            self.domain.name, created_by=self.web_user.username
+        )[0]
+        self.account.save()
```

Comment:
This ```.save()``` is not needed, because the django object returned by ```get_or_create_account_by_domain``` is already in sync with the database.